io.github.lxi-tools.yaml

```diff
@@ -0,0 +1,133 @@
+---
+app-id: io.github.lxi-tools.lxi-gui
+runtime: org.gnome.Platform
+runtime-version: '41'
+sdk: org.gnome.Sdk
+command: lxi-gui
+finish-args:
+  - "--share=ipc"
+  - "--socket=fallback-x11"
+  - "--socket=wayland"
+  - "--share=network"
+  - "--device=dri"
+  - "--filesystem=home"
+modules:
+  - name: lua
```

Comment:
Consider switching to https://github.com/flathub/shared-modules/tree/master/lua5.3.

Reply:
Done.